.dprintrc.json

```diff
@@ -10,10 +10,22 @@
   "markdown": {
     "textWrap": "always"
   },
-  "includes": ["**/*.{ts,tsx,js,jsx,json,md}"],
+  "includes": [
+    "**/*.{ts,tsx,js,jsx,json,md}",
+    "cli/dts/lib.deno.ns.d.ts",
+    "cli/dts/lib.deno.shared_globals.d.ts",
+    "cli/dts/lib.deno.unstable.d.ts",
+    "cli/dts/lib.deno.window.d.ts",
+    "cli/dts/lib.deno.worker.d.ts"
+  ],
```

Comment:
I think these `d.ts` files are included in the glob anyway.

Reply:
Yup :+1: